ykcapi/src/lib.rs

```diff
@@ -48,6 +52,99 @@ pub extern "C" fn __ykutil_get_llvmbc_section(res_addr: *mut *const c_void, res_
     }
 }
 
+struct Registers {
+    addr: *const usize,
+}
+
+impl Registers {
+    fn from_ptr(ptr: *const c_void) -> Registers {
+        Registers {
+            addr: ptr as *const usize,
+        }
+    }
+
+    fn read_from_stack(&self, off: isize) -> usize {
+        unsafe {
+            ptr::read::<usize>(self.addr.offset(off))
+        }
+    }
+
+    fn get(&self, id: u16) -> usize {
```

Comment:
Presumably this function (and maybe others?) should be `#[cfg(target_arch = "x86_64")]`.

Reply:
Fixed in a8383f5